src/REPLICA/neb.cpp

```diff
@@ -166,8 +168,12 @@ void NEB::command(int narg, char **arg)
     } else if (strcmp(arg[iarg],"verbose") == 0) {
       verbose=true;
       ++iarg;
+    } else if (strcmp(arg[iarg],"terse") == 0) {
```

Comment:
This is an undocumented addition. No feature gets added to LAMMPS without documentation.
Furthermore, the same situation as above applies here. This turns a binary choice into a ternary, so we should have an enumerator with symbolic constants (VERBOSE, NORMAL, TERSE).

But before even going there, please give the rationale for the "terse" option.

Reply:
I am not opposed to this kind of change. Having only summary output and skipping per-replica output is a sufficient justification for a third output verbosity option. I just wanted to know that and also it needs to be documented.